lib/src/utils.dart

```diff
@@ -43,45 +48,37 @@ abstract class ValueType {
   Value toValue();
 }
 
-extension NullValueUtils on Null {
-  Value toValue() {
-    return Value(nullValue: this);
-  }
-}
-
-extension NumValueUtils on num {
-  Value toValue() {
-    return Value(numberValue: toDouble());
-  }
-}
-
-extension StringValueUtils on String {
-  Value toValue() {
-    return Value(stringValue: this);
-  }
-}
-
-extension BoolValueUtils on bool {
-  Value toValue() {
-    return Value(boolValue: this);
-  }
-}
-
+// TODO fix To toValue() for List<T?>
 extension ListValueUtils<T extends ValueType> on List<T> {
   Value toValue() {
-    return Value(listValue: ListValue(values: this.map((e) => e.toValue())));
+    return Value(listValue: ListValue(values: map((e) => e.toValue())));
   }
 }
 
 extension MapStructUtils on Map<String, dynamic> {
-  Struct toStruct({bool skipErrors = false}) {
```

Comment:
Any reason to not let user's ignore errors?

Reply:
no particular reason just didn't understand why that would be an option, I meant to ask you why it was included so I'm glad you saw this. 

Should I add it back?

Reply:
Nah it's probably fine without it. We can always add it back later